Description: Change dependency from lcms to lcms2
Author: Tobias Frost <tobi@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/hsmagick.cabal
+++ b/hsmagick.cabal
@@ -23,7 +23,7 @@
 ghc-options:         -Wall
 extra-libraries:     tiff jasper jpeg png wmflite bz2 z m
 pkgconfig-depends:   GraphicsMagick >= 1.3.3
-                   , lcms
+                   , lcms2
                    , freetype2
                    , libxml-2.0
